README.md

```diff
@@ -1,2 +1,75 @@
-# Hello-Microverse
-My first Microverse project
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Pull Request should have a descriptive title so it will be easy to tell what changes are inside and a short summary describing introduced changes in more detail. This also makes your project documentation professional.

Reply:
Hi @ZeenatLawal

Thank you for your feedback. I have edited and updated the title of my PR but I can't seem to find the description box in order to add a short summary.

Reply:
Hello @Code-panther, your description box is this
![Screenshot (264)](https://user-images.githubusercontent.com/46851023/133332588-b76280c0-7446-41ab-8864-ebb49c53329b.png)
You can click on the ellipsis right next to the emoji icon to edit it 👍🏼

Reply:
Oh sorry, I think I found it and have provided the summary as per your suggestion.

Reply:
Glad you figured it out, you can go ahead and request another review now. 👍🏼

Reply:
Thank you.